client/layout/sidebar/custom-icon.jsx

```diff
@@ -30,12 +29,13 @@ const SidebarCustomIcon = ( { alt, className, icon, ...rest } ) => {
 			/>
 		);
 	}
-	if ( icon.indexOf( 'data:image' ) === 0 ) {
+	if ( icon.indexOf( 'data:image' ) === 0 || icon.indexOf( 'http' ) === 0 ) {
```

Comment:
Any chance to have `http` as part of the icon name in the future? Should we instead check if the icon starts with `http`?

Reply:
Hey @tyxla! 
This is an effort in accepting all the icon formats wordpress core currently accepts as menu icons `URLs, SVGs, and dashicon slug`. That said, checking only `icon.indexOf( 'http' ) === 0 ` would not account for `SVGs`.